docs/code-quality/ca1801.md

```diff
@@ -35,6 +35,10 @@ This rule does not examine the following kinds of methods:
 
 - Methods used as event handlers.
 
+- Serialization constructors ([see guidelines](https://docs.microsoft.com/fr-fr/dotnet/standard/serialization/serialization-guidelines#supporting-runtime-serialization)).
+
+- Serialization <xref:System.Runtime.Serialization.ISerializable.GetObjectData> method.
```

Comment:
Please review the syntax.

Reply:
Looks like you already fixed these and removed "fr-fr".

Reply:
xref looks fine